<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAppDialog</class>
 <widget class="QDialog" name="AddAppDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxCat">
     <property name="title">
      <string>Categories</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushAdd">
        <property name="text">
         <string>Add category</string>
        </property>
        <property name="icon">
         <iconset theme="list-add"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete category</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="listWidgetCategories"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelCommand">
     <property name="text">
      <string>Command:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <layout class="QGridLayout" name="buttonBar" columnstretch="0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>127</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Save Changes</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok">
         <normaloff/>
        </iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QPushButton" name="buttonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>127</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Quit dialog</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="window-close">
         <normaloff/>
        </iconset>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEditCommand"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="selectCommand">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelComment">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelIcon">
     <property name="text">
      <string>Icon:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="pushChangeIcon">
     <property name="text">
      <string>Set icon</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditComment"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkTerminal">
     <property name="text">
      <string>Run in Terminal</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="checkStartup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Startup Notifier</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
